src/main/webapp/feedback.jsp

```diff
@@ -0,0 +1,74 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%
+  boolean canOpen = (boolean) request.getAttribute("feedbackOpen");
+  String role = (String) request.getAttribute("role");
+  pageContext.setAttribute("feedbackOpen", canOpen);
+  pageContext.setAttribute("role", role); 
+%>
+
+
+<c:choose>
+  <c:when test= "${canOpen == false}">
+    <h2 style="text-align: center">You may not submit feedback yet.</h2>
+  </c:when>
+  <c:otherwise>
+    <c:choose>
+      <c:when test= "${role == 'Interviewee'}">
+        <h1 class="text-center">Please submit your feedback for your interviewer below</h1>
+        <!-- TODO: Send email on submit -->
+        <form action="/action_page.php" style="text-align:center">
+          <h5>Please Enter a value between 1 and 10 (1 being strongly disagree and 10 being strongly agree)</h5>
+          <label for="question1">I was comfortable during the interview:</label><br>
```

Comment:
Personally l don't like how everything is centered. l think left align is more professional. Please change it. But if Meagan asked you to change it to center from left align l don't want to make you do even more work so we can leave it as center.

Reply:
Yea Meagan suggested everything being centered

Reply:
It looked very bulky to me, seeing all the questions and input boxes to the left. I'd be open to maybe left aligned questions and right aligned inputs (for rating ones) or something like that. If all left-align is what you want, then that's fine too.

Reply:
I still like the centering of the buttons and no access messages though. Mainly to be consistent with everywhere else on the site.

Reply:
After discussion having form contents to the left and buttons centered is best

Reply:
l like centered buttons. And right aligned inputs l think would make it not clunky.

Reply:
Right now the scaled questions only accept numbers 1-10, no other values. An error message is shown when a number other than 1-10 is entered

Reply:
Input on same line as question / prompt looks much better. Thanks for working it out!